pkg/kudoctl/packages/resolver/resolver_incluster.go

```diff
@@ -0,0 +1,43 @@
+package resolver
+
+import (
+	"fmt"
+
+	kudoapi "github.com/kudobuilder/kudo/pkg/apis/kudo/v1beta1"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/packages"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/packages/convert"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/util/kudo"
+)
+
+// InClusterResolver resolves packages that are already installed in the cluster on the client-side. Note, that unlike
+// other resolvers, the resulting 'packages.Package' struct does not contain package 'packages.Files' (we don't have
+// the original files).
+type InClusterResolver struct {
```

Comment:
This is a second `InClusterResolver`: the [first one is on the server-side](https://github.com/kudobuilder/kudo/blob/main/pkg/controller/instance/resolver_incluster.go). Different k8s clients that we use on the server/client-side make it hard to consolidate. I'm not very proud of it but haven't found a better solution.